templates/UmbracoPackage/.template.config/dotnetcli.host.json

```diff
@@ -2,12 +2,16 @@
   "$schema": "https://json.schemastore.org/dotnetcli.host.json",
   "symbolInfo": {
     "Framework": {
-      "longName": "Framework",
-      "shortName": "F"
+      "longName": "framework",
+      "isHidden": true
     },
     "UmbracoVersion": {
       "longName": "version",
-      "shortName": "v"
+      "shortName": ""
+    },
```

Comment:
The only allowed framework version is `net7.0` (for now), so there's no need to show this parameter.

Since this value isn't supported on older Umbraco versions, any unaltered command would result in an `...is not a valid value...` error, so I've also updated the capitalization to align with the default ASP.NET Core templates.

Similarly, .NET 7 introduced a new [verbosity option](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new#options), taking the `v` short name, resulting in having to use `-p:v` for the Umbraco version instead. To remove any confusion, only allowing `--version` seems like a better idea.

Reply:
I just fear if the change in long name is breaking anyones buildscripts. That change should IMO only be for v12

Reply:
@ronaldbarendse I agree with Bjarke here, could you make a separate PR for this change targeting v12? I know we already bumped this to v11, so lets get it done now, so we don't keep pushing this change back 😁